cleantalk.php

```diff
@@ -626,6 +626,8 @@ function apbct_activation__new_blog($blog_id, $_user_id, $_domain, $_path, $_sit
     require_once(CLEANTALK_PLUGIN_DIR . 'inc/cleantalk-public.php');
     require_once(CLEANTALK_PLUGIN_DIR . 'inc/cleantalk-public-integrations.php');
 
+    apbct_form__get_noCookie_data();
```

Comment:
need a comment. mixing camelcase and snake_case. there is a word to get, but the result does not return anywhere.

Reply:
Done.